src/ACadSharp/Objects/MultiLeaderAnnotContext.cs

```diff
@@ -7,15 +8,17 @@
 using CSMath;
 
 
-namespace ACadSharp.Objects
-{
+namespace ACadSharp.Objects {
 
 	/// <summary>
 	/// This class represents a subset ob the properties of the MLeaderAnnotContext
 	/// object, that are embedded into the MultiLeader entity.
 	/// </summary>
 	public partial class MultiLeaderAnnotContext : NonGraphicalObject
```

Comment:
`MultiLeaderAnnotContext` should not inherit `NonGraphicalObject` because is not a `CadObject` with an assignable handle.

Reply:
Nevermind, leave the `NonGraphicalObject` and the AssignDocument() to make it simple, I'll think of a refactor after the merge, I don't want to get stuck on this particular issue.